test/views/CalculationViewSpec.scala

```diff
@@ -300,5 +296,77 @@ class CalculationViewSpec extends SpecBase with ViewBehaviours with ViewUtils {
 
       behave like pageWithExpectedMessages(expectedContent)(docWithOnlyOneCalculation)
     }
+
+    "it is an additional penalty and the user is an Agent (no estimate)" must {
+      def applyView(): HtmlFormat.Appendable = {
+        calculationAdditionalPage.apply(
+          isEstimate = false,
+          startDate = "1 April 2022",
+          endDate = "30 June 2022",
+          dueDate = "17 October 2022",
+          penaltyAmount = "50.50",
+          amountReceived = "40.10",
+          amountLeftToPay = "10.40")(implicitly, implicitly, implicitly, agentUser)
+      }
+
+      implicit val doc: Document = asDocument(applyView())
+
+      val expectedContent = Seq(
+        Selector.title -> titleLPP,
+        Selector.periodWithText -> periodWithText,
+        Selector.HeaderTextNotVisible -> periodHiddenText,
+        Selector.h1 -> headingLPP,
+        Selector.howPenaltyIsApplied -> howPenaltyIsAppliedLPP2,
+        Selector.whenPenaltyIncreases -> whenPenaltyIncreases,
+        Selector.calculation -> lpp2Calculation,
+        Selector.summaryListRowKey(1) -> dueDate,
+        Selector.summaryListRowValue(1) -> "17 October 2022",
+        Selector.summaryListRowKey(2) -> th2LPP,
+        Selector.summaryListRowValue(2) -> "£50.50",
+        Selector.summaryListRowKey(3) -> th3LPP,
+        Selector.summaryListRowValue(3) -> "£40.10",
+        Selector.summaryListRowKey(4) -> th4LPP,
+        Selector.summaryListRowValue(4) -> "£10.40",
+        Selector.link -> link
+      )
+
+      behave like pageWithExpectedMessages(expectedContent)
+    }
+
+    "it is an additional penalty and the user is an Agent (estimate)" must {
```

Comment:
"it is an additional penalty and the user is an Agent (estimate)" can be "it is a second payment penalty and the user is an Agent (estimate)"